python/cudf/cudf/_libxx/lib.pxd

```diff
@@ -0,0 +1,146 @@
+        column_view& operator=(column_view&&)
+        column_view(data_type type, size_type size, const void* data)
+        column_view(data_type type, size_type size, const void* data,
+                    const bitmask_type* null_mask)
+        column_view(data_type type, size_type size, const void* data,
+                            const bitmask_type* null_mask, size_type null_count)
+        column_view(data_type type, size_type size, const void* data,
+                            const bitmask_type* null_mask, size_type null_count,
+                            size_type offset)
+        column_view(data_type type, size_type size, const void* data,
+                            const bitmask_type* null_mask, size_type null_count,
+                            size_type offset, vector[column_view] children)
+        T* data[T]()
+        bitmask_type* null_mask()
+        size_type size()
+        data_type type()
+        bool nullable()
+        size_type null_count()
+        bool has_nulls()
+        size_type offset()
+        size_type num_children()
+
+    cdef cppclass mutable_column_view:
+        mutable_column_view()
+        mutable_column_view(const mutable_column_view&)
+        mutable_column_view& operator=(const mutable_column_view&)
+        mutable_column_view(data_type type, size_type size, const void* data)
+        mutable_column_view(data_type type, size_type size, const void* data,
+                            const bitmask_type* null_mask)
+        mutable_column_view(data_type type, size_type size, const void* data,
+                            const bitmask_type* null_mask, size_type null_count)
+        mutable_column_view(data_type type, size_type size, const void* data,
+                            const bitmask_type* null_mask, size_type null_count,
+                            size_type offset)
+        mutable_column_view(data_type type, size_type size, const void* data,
+                            const bitmask_type* null_mask, size_type null_count,
+                            size_type offset, vector[mutable_column_view] children)
+        T* data[T]()
+        bitmask_type* null_mask()
+        size_type size()
+        data_type type()
+        bool nullable()
+        size_type null_count()
+        bool has_nulls()
+        size_type offset()
+        size_type num_children()
+        
+cdef extern from "cudf/table/table_view.hpp" namespace "cudf" nogil:
+    cdef cppclass table_view:
+        table_view()
+        table_view(const vector[column_view])
+        column_view column(size_type column_index)
+        size_type num_columns()
+        size_type num_rows()
+
+    cdef cppclass mutable_table_view:
+        mutable_table_view()
+        mutable_table_view(const vector[mutable_column_view])
+        mutable_column_view column(size_type column_index)
+        size_type num_columns()
+        size_type num_rows()
+
+cdef extern from "cudf/table/table.hpp" namespace "cudf::experimental" nogil:
```

Comment:
is this still in the experimental namespace?

Reply:
Yes